### DISCUSSION TITLE
why-netdata-is-free/

### DISCUSSION THREAD

**Discussion by giscus** [netdata/blog#172] [General]
# why-netdata-is-free/

Why Netdata is free

https://blog.netdata.cloud/why-netdata-is-free/

**Comment by lenainjaune** [comment]
Hello,

I am a French system administrator and take care of the computer park of a small company. Recently I had a problem with our backup server being stuck after BIOS. While investigating I noticed that once the system started sometimes the machine put itself in a situation of stress and made a hell of a noise (the fans started to blow hard). Since the problem was not continuous and I didn't know where to look, I wanted to monitor fan activity so I could correlate it to system logs. So I looked for a surveillance solution that was quick to set up, light and efficient. 

I heard about Netdata and on Github the remark "People get addicted to Netdata. Once you use it on your systems, there's no going back! You've been warned..." amused and intrigued me. So I wanted to experience it. Result: installation on the server in a few seconds, immediate access to the web interface in HTTP via port 19999, no configuration to do, all the activities of the components of the machine reassembled... stunning :smile:

Here I am in the process of learning it and therefore I am looking for information. And I came across this article and it really touched me. In particular the vision that you present of the Open Source world in order to publish for the commons (thank you for sharing your passion, your time and your work :wink: ) and especially that things take time. The time they need. From you to me, fed up with these hastily made software, which pretty much works. All this for what ? For short-term profitability, for a disposable product immediately replaced by another equally disposable product. 

The proverb says: 
> Rome was not built in a day

I think some new technology players should think about it... 

I wish you make it as awesome as you want and long life to this project :smile: 

With adelphity
lnj
